wiki/Tournaments/CJB/2018/4K/en.md

```diff
@@ -40,33 +40,67 @@ Prizes for this tournament are donated by ![][flag_CN] [hans1999](https://osu.pp
 
 ## Links
 
-- [Livestream](https://live.bilibili.com/5333)
+- [Livestream of Rasis](https://live.bilibili.com/5333)
+- [Livestream of Flow Light](https://live.bilibili.com/73668)
+- [Livestream of aiyulu](https://live.bilibili.com/10183)
 - [Tournament QQ Group](https://jq.qq.com/?_wv=1027&k=5sTdKLp)
 - [Challonge](https://challonge.com/zh_CN/cjb4)
 
 ## Match Schedule
 
-### Semifinals
+Mappool of _Finals - Week 2_ will be shown at 2018-08-10 15:00 (UTC+8).
+
+### Finals - Week 1
 
-#### Saturday, 4. August 2018
+#### Saturday, 11. August 2018
 
 | Match Time (UTC+8) | Player 1 | | | Player 2 | Match ID |
 | :--: | ---: | ---: | :--- | :--- | :--: |
```

Comment:
The delimiter suppose to follow these:
- `---` (for left align)
- `:--` (for left align)
- `--:` (for right align)
- `:-:` (for centre align)
do for others too

Reply:
Consistent formatting for left align. We use `:--` as the main format.